google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -147,133 +138,75 @@ def hocr_bounding_box(self):
 
 
 @dataclasses.dataclass
-class Line:
-    """Represents a wrapped documentai.Document.Page.Line.
+class Symbol(PageElement):
+    """Represents a wrapped documentai.Document.Page.Symbol.
+    https://cloud.google.com/document-ai/docs/process-documents-ocr#enable_symbols"""
 
-    Attributes:
-        documentai_object (google.cloud.documentai.Document.Page.Line):
-            Required. The original google.cloud.documentai.Document.Page.Line object.
-        _page (Page):
-            Required. The Page object.
-    """
+    @property
+    def hocr_bounding_box(self):
+        return None
```

Comment:
Does this mean that symbols are not represented in the hocr format?  If so please add some code comments here to document that.

Same comment about math formula below.